pyEpiabm/pyEpiabm/tests/test_routine/test_space_forces.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+
+import pyEpiabm as pe
+from pyEpiabm.property import InfectionStatus
+from pyEpiabm.routine import SpatialInfection
+
+
+class TestSpatialInfection(unittest.TestCase):
+    """Test the 'PlaceInfection' class, which contains the
+    infectiousness and susceptibility calculations that
+    determine whether infection events occur within places.
+    Each function should return a number greater than 0.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
+        """Intialise a population with one infector and one
+        infectee, both in the same place and household.
+        """
+        cls.cell = pe.Cell()
+        cls.microcell = pe.Microcell(cls.cell)
+        cls.infector = pe.Person(cls.microcell)
+        cls.infectee = pe.Person(cls.microcell)
+        cls.timestep = 1
+        pe.Parameters.instance().basic_reproduction_num = 2.8
+
+    def test_space_susc(self):
+        result = SpatialInfection.space_susc(self.cell, self.infectee,
+                                             self.timestep)
+        self.assertTrue(result > 0)
+        self.assertIsInstance(result, float)
+
+    def test_space_inf(self):
+        result = SpatialInfection.space_inf(self.cell, self.infector,
+                                            self.timestep)
+        self.assertTrue(result > 0)
+        self.assertIsInstance(result, float)
+
+    def test_space_foi(self):
```

Comment:
Do we require this to be less than one as well, or can it be greater than 1 too (resulting in guaranteed infection)?

Reply:
I like allowing it to be any non-negative number, as I feel an infectiousness of 3 is very different to one of 1, even though they both lead to the same number of people infected.

Reply:
I like allowing it to be any non-negative number, as I feel an infectiousness of 3 is very different to one of 1, even though they both lead to the same number of people infected.

Reply:
I am now convinced that the force of infection may be greater than 1 and so it may be beneficial to allow this in the future!